src/bugreporter/odemis_bugreporter.py

```diff
@@ -130,7 +130,7 @@ def parse_config(configfile):
     return config
 
 
-class OdemisBugreporter():
+class OdemisBugreporter:
```

Comment:
No, in Python 2, we (still) need to write `class OdemisBugreporter(object)`. Without inheriting from object, it also works, but you the class is an "old-style" class, with weird behaviour.